pricing-page-exercise/dilara/styles.css

```diff
@@ -0,0 +1,234 @@
+.nav-bar {
```

Comment:
First thing, try to always include this code at the very top of your css file:

```

* {
  box-sizing: border-box; 
}
```

This would help avoid unexpected behaviour with element sizing. See Lesson 3 notes: https://classroom.google.com/c/NTg0NzkwMzQ4MjQ2/p/NTkyOTU1NjY3NTEz/details